### PR TITLE
Adding project_id to the database payload

### PR DESCRIPTION
**Pull request by rpmoore**

### CODE REVIEW

**Comment by ChiefMateStarbuck** [comment]
!deploy

File: specification/resources/databases/models/database_cluster.yml [hunk 1]
```diff
@@ -103,6 +103,11 @@ properties:
     allOf:
     - $ref: './database_maintenance_window.yml'
     - readOnly: true
+  project_id:
+    type: string
+    format: uuid
+    example: 9cc10173-e9ea-4176-9dbc-a4cee4c4ff30
+    description: The id of the project the database cluster is currently in.
```
Comment (andrewsomething) [inline specification/resources/databases/models/database_cluster.yml:110]: "is currently in" - Is this a read only attribute or can it be used to set the project at create time?

Comment (rpmoore) [reply specification/resources/databases/models/database_cluster.yml:110]: It can be set at cluster creation.

Comment (rpmoore) [reply specification/resources/databases/models/database_cluster.yml:110]: Sure think, I'll update it and push.

**Review by ChiefMateStarbuck — APPROVED** [review]

**Review by andrewsomething — APPROVED** [review]
:+1: lgtm!

**Pull request merged**